beaconChain/tests/helpers/stateTransitionHelpers.test.ts

```diff
@@ -323,3 +324,59 @@ describe("isActiveValidator", () => {
     assert.isFalse(result);
   });
 });
+
+describe("getForkVersion", () => {
+  const fork: Fork = {
+    epoch: new BN(12),
+    previousVersion: new BN(4),
+    currentVersion: new BN(5)
+  };
+
+  const four: uint64 = new BN(4);
+  const five: uint64 = new BN(5);
+
+  it("fork version should be 4", () => {
```

Comment:
I might make the text relate to how the fork version is being selected.

eg: "epoch before fork epoch returns previous fork version",
"epoch after fork epoch returns current fork version", and
"epoch on fork epoch returns current fork version"

or something to that effect